### PR TITLE
fix: inconsistency in the user interface

### LINKED ISSUE #411: Inconsistency in the User Interface
**Describe the bug**
A substantial spacing can be identified between the title of each page and its contexts.

**To Reproduce**

1. Open up the Talawa Project Documentation website
2. Navigate to the developer resources section through the side bar
3. Click on the Talawa API section 
4. Open up the pages in this section 

**Expected behavior**
A consistent user interface that is easily understandable and aesthetically pleasing as seen [here](https://docs.talawa.io/docs/)

**Actual behavior**
Inconsistencies as the spacing between the header and title differs in each page
**Screenshots**
Screen Shot 2023-03-09 at 18 37 02
Screen Shot 2023-03-09 at 18 36 55
Screen Shot 2023-03-09 at 18 39 02

**Additional details**
None

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by faveee**
**What kind of change does this PR introduce?**
Refactoring

**Issue Number:** #411 

Fixes #411 

**Did you add tests for your changes?**
No

**Snapshots/Videos:**
Nil

**If relevant, did you update the documentation?**
Yes, i did

**Summary**
This was observed while going through the web page. Although seemingly small, it doesn't look decent for a professional website. The disparity is so glaring that it gives the web page an amateur feel thereby defacing the overall design of the website.

**Does this PR introduce a breaking change?**
No, it doesn't

**Other information**
Nil

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-docs/blob/develop/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

File: docs/developers/talawa-api/expectations.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ id: expectations
 title: Expectations
 ---
 
+> This page provides an in overview to what is expected from the Talawa API thereby increasing productivity and improving the application.
```
Comment (Cioppolo14) [inline docs/developers/talawa-api/expectations.md]: Please remove the word "in", and break this into two sentence. The second one beginning with, "Thereby,"

File: docs/developers/talawa-api/mutations.md [hunk 2]
```diff
@@ -3,10 +3,11 @@ id: mutations
 title: Mutations
 ---
 
+> Mutations can be classified in a number of ways, including by the type of change that they introduce, the part of the page that they affect, or the manner in which they are introduced. Talawa API introduces mutation to alter it's value to a certain degree.
+
 ## Mutations According to Roles
-This doucument outlines our mutations and to which type of user each is constrained
+This doucument outlines our mutations and to which type of user each is constrained. Common types of mutations include insertions and deletions among others. Insertions are new changes to the Talawa API, while deletions removes.
```
Comment (Cioppolo14) [inline docs/developers/talawa-api/mutations.md]: Please fix the legacy error in the spelling of document.  Please remove the words "among others"

**Review by Cioppolo14 — CHANGES_REQUESTED** [review]
Thank for your this submission! I have a few editorial comments before we're ready to merge.

**Comment by faveee** [comment]
Thank you for the feedback @Cioppolo14 . I apologise for the late response. I've added the requested changes, awaiting your review!

File: docs/developers/talawa-api/expectations.md (hunk 1 above)
Comment (faveee) [reply docs/developers/talawa-api/expectations.md]: @Cioppolo14 Can you kindly review this PR? Thank you!

**Comment by Cioppolo14** [comment]
@faveee Can you please resolve the conflicting files?

File: docs/developers/talawa-api/expectations.md [hunk 3]
```diff
@@ -3,6 +3,8 @@ id: expectations
 title: Expectations
 ---
 
+> This page provides an overview to what is expected from the Talawa API. Thereby increasing productivity and improving the application.
```
Comment (Cioppolo14) [inline docs/developers/talawa-api/expectations.md]: Please add the word "as" between overview and to.  For example, "This page provides an overview as to what is expected..."

**Review by Cioppolo14 — CHANGES_REQUESTED** [review]
Please fix the conflicting files and the one comment on the expectations file.  Thank you!

**Comment by faveee** [comment]
@Cioppolo14 In my bid to resolve the conflicting files I mistakenly closed this PR. Can you please reopen and revert?

I look forward to your response. Thank you

**Comment by faveee** [comment]
> @faveee Can you please resolve the conflicting files?

Hello! @Cioppolo14 All conflicting files have been resolved.

File: docs/developers/talawa-api/expectations.md (hunk 1 above)
Comment (Cioppolo14) [reply docs/developers/talawa-api/expectations.md]: @faveee Please add a , after Thereby

File: docs/developers/talawa-api/mutations.md (hunk 2 above)
Comment (Cioppolo14) [reply docs/developers/talawa-api/mutations.md]: @faveee  Please remove the words "among others"

**Review by Cioppolo14 — CHANGES_REQUESTED** [review]
Please address my comments I put in the responses to previous comments.

**Comment by faveee** [comment]
> Please address my comments I put in the responses to previous comments.

I apologize for the back and forth. I erroneously deleted the previous commits and closed the PR in my bid to resolve the conflicting files.

**Review by Cioppolo14 — APPROVED** [review]

File: docs/developers/talawa-api/mutations.md [hunk 4]
```diff
@@ -2,8 +2,16 @@
...
 title: Mutations
 ---
+> Mutations can be classified in a number of ways, including by the type of change that they introduce, the part of the page that they affect, or the manner in which they are introduced. Talawa API introduces mutation to alter it's value to a certain degree.
 
 ## Mutations According to Roles
+<<<<<<< HEAD
+=======
+
+This document outlines our mutations and to which type of user each is constrained. Common types of mutations include insertions and deletions. Insertions are new changes to the Talawa API, while deletions removes.
+
```
Comment (palisadoes) [inline docs/developers/talawa-api/mutations.md:10]: 1. Please fix this. The >>> indicate a merge conflict. @Cioppolo14 FYI
2. This file should be removed from the PR

**Review by palisadoes — CHANGES_REQUESTED** [review]
Please make the requested changes

File: docs/developers/talawa-api/mutations.md (hunk 4 above)
Comment (faveee) [reply docs/developers/talawa-api/mutations.md:10]: @palisadoes Are you requesting for the mutations.md file to be deleted or for the conflict to be resolved? I'm not quite clear on your instruction.

**Review by palisadoes — APPROVED** [review]

**Pull request merged**